### PR TITLE
Send a notification to CP members when a CA is created

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: src/CitizenProject/CitizenProjectManager.php [hunk 1]
```diff
@@ -28,6 +28,9 @@
 
 class CitizenProjectManager
 {
+    private const EXCLUDE_HOSTS = false;
```
Comment (thomasbisignani) [inline src/CitizenProject/CitizenProjectManager.php]: Can you use `Enum` to manage const please ?

File: src/Mailer/Message/CitizenActionNotificationMessage.php [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        CitizenAction $citizenAction,
+        string $citizenActionAttendLink
+    ): self {
+        if (!$recipients) {
+            throw new \InvalidArgumentException('At least one recipient is required.');
+        }
+
+        $recipient = array_shift($recipients);
+        if (!$recipient instanceof Adherent) {
+            throw new \InvalidArgumentException(sprintf('This message builder requires a collection of %s instances', Adherent::class));
```
Comment (thomasbisignani) [inline src/Mailer/Message/CitizenActionNotificationMessage.php]: CS : please write in multiple lines

File: src/Mailer/Message/CitizenActionNotificationMessage.php [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+            $recipient->getFullName(),
+            '[Projets citoyens] Une nouvelle action citoyenne au sein de votre projet citoyen !',
+            static::getTemplateVars($host, $citizenAction, $citizenActionAttendLink),
+            static::getRecipientVars($recipient),
+            $host->getEmailAddress()
+        );
+
+        foreach ($recipients as $recipient) {
+            if (!$recipient instanceof Adherent) {
+                throw new \InvalidArgumentException(sprintf('This message builder requires a collection of %s instances', Adherent::class));
```
Comment (thomasbisignani) [inline src/Mailer/Message/CitizenActionNotificationMessage.php]: Same here

**Comment by thomasbisignani** [comment]
:octocat:

File: src/Mailer/Message/CitizenActionNotificationMessage.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+            throw new \InvalidArgumentException(sprintf(
+                'This message builder requires a collection of %s instances',
+                Adherent::class
+            ));
+        }
+
+        $message = new static(
+            Uuid::uuid4(),
+            '326404',
+            $recipient->getEmailAddress(),
```
Comment (B-Galati) [inline src/Mailer/Message/CitizenActionNotificationMessage.php]: I know it's not you but it's a real pain not to manage one or multiple recipient directly in `Message`. I spend some time trying to understand while it should be straightforward.

One day we should fix that :-)

File: src/Repository/CitizenProjectMembershipRepository.php [hunk 5]
```diff
@@ -147,6 +147,17 @@ public function findPriviledgedMembers(string $citizenProjectUuid, array $privil
         return $this->createAdherentCollection($query);
     }
 
+    public function findFollowers(string $citizenProjectUuid, bool $includeAdministrators = true): AdherentCollection
+    {
+        $privileges = [CitizenProjectMembership::CITIZEN_PROJECT_FOLLOWER];
+
+        if ($includeAdministrators) {
+            $privileges = array_merge($privileges, [CitizenProjectMembership::CITIZEN_PROJECT_ADMINISTRATOR]);
```
Comment (B-Galati) [inline src/Repository/CitizenProjectMembershipRepository.php]: Simpler to do :

`$privileges[] = CitizenProjectMembership::CITIZEN_PROJECT_ADMINISTRATOR;`

File: tests/Controller/EnMarche/CitizenActionManagerControllerTest.php [hunk 6]
```diff
@@ -125,6 +126,9 @@ public function testCreateCitizenActionSuccessful()
         $this->assertInstanceOf(CitizenAction::class, $citizenAction = $this->getCitizenActionRepository()->findOneBy(['slug' => (new \DateTime())->format('Y-m-d').'-mon-action-citoyenne']));
         $this->assertSame('Mon Action Citoyenne', $citizenAction->getName());
         $this->assertCountMails(0, EventRegistrationConfirmationMessage::class, 'jacques.picard@en-marche.fr');
+        $this->assertCountMails(1, CitizenActionNotificationMessage::class, 'jacques.picard@en-marche.fr');
+        $this->assertCountMails(1, CitizenActionNotificationMessage::class, 'gisele-berthoux@caramail.com');
+        $this->assertCountMails(1, CitizenActionNotificationMessage::class, 'luciole1989@spambox.fr');
```
Comment (B-Galati) [inline tests/Controller/EnMarche/CitizenActionManagerControllerTest.php]: Let's add a random (hard-coded not really random) person(s) that is not supposed to receive this email ? Not sure it's really useful :D

File: src/Mailer/Message/CitizenActionNotificationMessage.php (hunk 4 above)
Comment (Remg) [reply src/Mailer/Message/CitizenActionNotificationMessage.php]: Clearly a thing that could be done in addition of #2228

File: tests/Controller/EnMarche/CitizenActionManagerControllerTest.php (hunk 6 above)
Comment (Remg) [reply tests/Controller/EnMarche/CitizenActionManagerControllerTest.php]: " it's free \o/ "

**Pull request merged**